docs/data/toolpad/core/components/page-container/page-container.md

```diff
@@ -47,7 +47,7 @@ The breadcrumbs contains **ACME / Home / Orders** when you visit the path **/hom
 
 {{"demo": "TitleBreadcrumbsPageContainer.js", "height": 300, "hideToolbar": true}}
 
-## Dynamic Routes
+## Dynamic routes
```

Comment:
@bharatkashyap Similar to #4306, I fixed a couple more to match https://www.notion.so/mui-org/Writing-style-guide-2a957a4168a54d47b14bae026d06a24b.

cc @samuelsycamore we are progressively getting to match the style guide everywhere 😄